### ISSUE TITLE
[Campaign Landing Page] [UX] In the editor experience, some sections are missing the' required asterisk' that denotes a required field

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-cms#19945]
## Description

This is a Drupal/Editor-facing issue. In ticket #19163 , this issue was resolved specifically for the Spotlight field, but it was discovered during implementation testing that other sections are also missing the 'required field' red asterisk, causing a confusing editor experience when they are creating/editing/saving Campaign Landing Pages.

The following sections are missing this attribute:

- Video
- What you can do - Promos
- Page-specific or Reusable Q&As

## Tasks
- Determine all instances in Campaign Landing Page editor experience where the required asterisk is missing and list in this ticket.
- Provide alternative error text for any confusing error messages.
- Providing error text for any missing error messages

### Additional Details

- Video section: The Add Media button should  have the red asterisk. _UX Question: Since a minimum of 2 are required, should there be help text under the button, or displayed after the button is clicked?_
![Image](https://github.com/user-attachments/assets/189422a0-ac68-4eef-90e6-5870280eefb4)

- What you can do Promos
![Image](https://github.com/user-attachments/assets/4072530d-6597-4b8e-b364-b76fea167d43)

- Page-specific or Reusable Q&As
![Image](https://github.com/user-attachments/assets/13623d98-c52d-4f1f-9d2a-7a56ab797541)

### User story

**AS A** Campaign Landing Page editor/creator
**I WANT** to know at a glance which fields are required
**SO THAT** I'm not frustrated when I leave fields blank, click save or publish, and then am confronted with errors of fields being required that don't indicate they are required.

### Roles / assignments
_After functional testing, code review, accessibility review, and design review can happen in parallel._
- [ ] **Functional testing:** 
- [ ] **Code review:** 
- [ ] **Design review:** 
- [ ] **Accessibility review:** Jacqueline 
- [ ] **Product acceptance:** Fran Cross

## Acceptance criteria

- [ ] All fields that are required, but don't have the required asterisk, are listed in this ticket
- [ ] Tag Fran Cross so she can create a Drupal engineering dev ticket
- [ ] Okay to close ticket.

**Comment by jilladams** [comment]
There are a couple UX questions nested in here before we could assign it to a Drupal engineer. When we refine, we can sort out if Dave or Jordan can give insights for help text etc on the fly or would prefer a separate ticket?

**Comment by jilladams** [comment]
(In the spirit of letting Product own the To Refine column / Refinement agenda, I didn't bump this ticket to that column.)

**Comment by FranECross** [comment]
Sounds good; I'm actually still grooming/updating the ticket, so will move it over soonish when I finish up. :)
